controls/radchartview/howto/hide-zerro-values-pieseries.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Hide Zero Values Labels in PieChart
+page_title: Remove Zero Values Labels in PieChart
+description: This article described how to hide zero segments labels in PieChart
+slug: radchartview-troubleshooting-hide-zero-segments-labels
+tags: troubleshooting,howto,zero,piechart,label,pieseries,slice,hide,remove,segments
+published: True
+position: 0
+---
+
+# Hide Zero Segments Labels in PieChart 
+
+There's different ways to hide/remove the labels of the zero segments inside __PieChart__. This article will describe one of them. In a few words we are going to hide the labels by binding the __Opacity__ property of the TextBlock which holds it. For the purpose of this example, firstly we will create the following ViewModels which are use for both chart scenarios:
+
+#### __[C#] Example 1: Creating ViewModels__
+{{region radchartview-troubleshooting-hide-zero-segments-labels_0}}	
```

Comment:
In my opinion, we can simplify the approach shown here.

Now we have a view model definition, population of the items and DataContext setting. Also, there is an Opacity property that is used only for the 0 values. I think this shifts the focus of the article.

Instead consider to use a simpler example. For example, a RadPieChart with several PieDataPoints in XAML. And an IValueConverter that bounds to the Value property of the PieDataPoint in the Template of the label definition. This approach will be the same in both MVVM and non-MVVM scenarios.